packages/bsky/tests/views/blocks.test.ts

```diff
@@ -595,4 +597,15 @@ describe('pds views with blocking', () => {
     const combined = [...first.data.blocks, ...second.data.blocks]
     expect(combined).toEqual(full.data.blocks)
   })
+
+  it('returns knownFollowers with blocks filtered', async () => {
+    const carolForAlice = await agent.api.app.bsky.actor.getProfile(
+      { actor: bob },
+      { headers: await network.serviceHeaders(alice) },
+    )
+
+    const knownFollowers = carolForAlice.data.viewer?.knownFollowers
+    expect(knownFollowers?.count).toBe(1)
+    expect(knownFollowers?.followers).toHaveLength(0)
```

Comment:
Just noting here `count` is 1, but the blocked user has been filtered out. We return 5 to the client, and the client uses only 3, so we're already overfetching. Discussed with Devin and decided this was a rare enough case to leave in for now. Frontend will handle the difference.